src/NuGet.Clients/NuGet.VisualStudio.Common/Experimentation/NuGetExperimentationService.cs

```diff
@@ -29,10 +29,17 @@ internal NuGetExperimentationService(IEnvironmentVariableReader environmentVaria
 
         public bool IsExperimentEnabled(ExperimentationConstants experimentation)
         {
-            var isEnvVarEnabled = !string.IsNullOrEmpty(experimentation.FlightEnvironmentVariable)
-                && _environmentVariableReader.GetEnvironmentVariable(experimentation.FlightEnvironmentVariable) == "1";
+            bool isExpForcedEnabled = false;
+            bool isExpForcedDisabled = false;
```

Comment:
@donnie-msft Did you mean implicit type or maybe something else? I am currently using an explicit type but since it's assigning false, are you suggesting an implicit type instead?

Reply:
Yep! Please see rule # 10 https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#c-coding-style